_data/glossary/podpreset.yaml

```diff
@@ -0,0 +1,12 @@
+id: podpreset
+name: PodPreset
+tags:
+- operation
+short-description: >
+  An API object that injects information such as secrets, volume mounts,
+  and environment variables into pods at creation time.
+long-description: >
+  This object chooses the pods to inject information into using
+  standard selectors. This allows the podspec definitions to be
```

Comment:
Better add a link to "selector" term.
"podspec" is a weird term, please consider revise.